src/features/defi/providers/fox-farming/components/FoxFarmingManager/Deposit/components/Confirm.tsx

```diff
@@ -143,6 +197,8 @@ export const Confirm: React.FC<StepComponentProps & { accountId: AccountId | und
         },
         assets,
       )
+
+      navigate(-1)
```

Comment:
Didn't test it at runtime yet but hard to understand while reading if this is right or not, can we have an approval step ro something similar that would be wrong in term of navigation? 

Will check this out during runtime test